Naga female decals & new textures

kaldrskegg opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
Added new diffuse, normals and properties for all naga female parts
Blendshapes for naga female
New tail for naga female
New ethnicity stuff for Naga Female

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
Added naga_body, naga_head & naga_attachment_diffuse, _properties & _normal
Added temporary gene files for testing these new additions as wc_gene_naga_body_test and naga_head_test
Added changes to female_body.asset, morph_genes.txt and elven ethnicities txt file.
Added blendshapes for no ears, changed nose and rotating eyes
<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
Boot in debug mode, open portrait editor, turn on genes naga_body_test and naga_head_test.

kaldrskegg committed:
new diffuses,properties,normals

this shit is a bit all over the place right now, but I'm making a new branch to get some help on the working the decals out.

zumbak04 committed:
Nage Ethnicity Decals

Applied new decals to ethnicity.

zumbak04 committed:
Portrait Editor Crash

Fixed the Portrait Editor crash.

zumbak04 committed:
Double blend_modes

Fixed 2 entrances of blend_modes.

zumbak04 committed:
Update wc_ethnicities_elven.txt

(GDC)

kaldrskegg committed:
Merge branch 'master' into naga-female-decals-&-new-textures

zumbak04 committed:
portrait_skin_attachment_alpha_to_coverage

Created a new shader that doesn't apply decals but apply skin color: portrait_skin_attachment_alpha_to_coverage

zumbak04 committed:
Merge branch 'naga-female-decals-&-new-textures' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2 into naga-female-decals-&-new-textures

zumbak04 commented on common/coat_of_arms/coat_of_arms/wc_easter_egg_dynasties.txt:
```diff
@@ -0,0 +1,10 @@
+### Humans ###
```
The properties file is never yellow. How they work is explained in the Great Big Guide.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Model-Guidelines#guidelines
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126911951-deff94fd-f1d6-4931-84ec-3f5455a8e6a0.png)

</details>

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female' into naga-female-decals-&-new-textures

zumbak04 committed:
No Red Channel

Removed red channel from properties.
(GDC)

kaldrskegg commented on common/ethnicities/wc_ethnicities_elven.txt:
```diff
@@ -973,4 +973,7 @@ naga_ethnicity = {
 	gene_grade = {
```
its the issue you described before, because the webbing is a 2d mesh. I will try duplicating it so it has front faces on both sides and see if it helps.

kaldrskegg replied:
fixed a previous issue where a mesh had some quads causing lighting problems, + fixed some UV issues. Game does not like double vertexes tho in some instances
![image](https://user-images.githubusercontent.com/46745564/126917503-b764aa80-336d-45b7-a1eb-ca23c0fb09b7.png)

kaldrskegg replied:
![image](https://user-images.githubusercontent.com/46745564/126917783-94ff354d-d562-445d-a513-7e1a263cfe3d.png)
all backfaces are really shiny

kaldrskegg committed:
mesh changes, fixed quads

fixed lighting issue from Quads in original mesh on back fins. Fixed some UV issues on head fins.

kaldrskegg committed:
new tail + webbing mesh change + blendshapes

made blendshapes for rotating the eye without the eyeball (may need to make an animation to match),and blendshape to remove ear

kaldrskegg committed:
naga nose blendshape

kaldrskegg committed:
blendshape code

an attempt was made

kaldrskegg committed:
naga ethnicity

zumbak04 committed:
Naga Belly Color

Naga use hair color on their belly.

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female' into naga-female-decals-&-new-textures

# Conflicts:
#	gfx/FX/jomini/portrait.shader

zumbak04 committed:
Moved Genes to Indexes

Moved genes to free indexes.

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female' into naga-female-decals-&-new-textures

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female-decals-&-new-textures' into naga-fixes

zumbak04 committed:
naga_skin to complexion

Moved naga_skin to complexion (since it's supposed to replace other skin variations).

zumbak04 committed:
Blocked Naga Clothes

Blocked nagas from having hairs, clothes in the Ruler Designer and in-game.

zumbak04 committed:
Naga Gene Errors

Fixed gene indexes and missing attributes.

zumbak04 reviewed (changes requested):
Don't try to fix this before #560 merged. This branch has major bugs that won't allow you to reproduce the issues I'm talking about.
Plus, if you don't know how to fix these bugs, I can tell you how.

zumbak04 committed:
No By Default

Made it so other ethnicities don't have naga noses, eyes and ears by default.

zumbak04 committed:
Moving Shader
(GDC)

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female' into naga-female-decals-&-new-textures

zumbak04 committed:
Merge pull request #560 from Warcraft-GoA-Development-Team/naga-fixes

Naga Fixes

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female-decals-&-new-textures' into naga-belly-color

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/naga-female-decals-&-new-textures' into naga-belly-color

# Conflicts:
#	common/genes/wc_gene_body_naga_test.txt

zumbak04 committed:
PS_skin_attachment Gone

Removed PS_skin_attachment and replaced it by DECALS define.

zumbak04 committed:
Shader Instead of Tail Decal

Replaced the naga tail decal by a shader using the red channel of the properties.

zumbak04 committed:
Merge pull request #559 from Warcraft-GoA-Development-Team/naga-belly-color

Naga Belly Color

kaldrskegg committed:
changes to model 2021

work i did on the model in 2021, properties, some temp files in here, i did diffuses and some texture work. lost track of what i was doing at the time
